test/test_cpp_extensions.py

```diff
@@ -100,6 +100,11 @@ def test_jit_cuda_extension(self):
         # 2 * sigmoid(0) = 2 * 0.5 = 1
         self.assertEqual(z, torch.ones_like(z))
 
+    @unittest.skipIf(not TEST_CUDA, "CUDA not found")
+    def test_cuda_rng_multithread(self):
+        import torch_test_rng
+        torch_test_rng.cuda_rng_multithread()
```

Comment:
This is meant to test C++ extensions, and not as a way to write general tests in C++. Please do it differently.